Quickstarts

khorne3 opened the pull request:
* [Homepage](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/)
* [Linux/macOS](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/quickstarts/linux-macos)
* [Windows](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/v/quickstart/quickstarts/windows)

vlad-alexandru-ionescu commented on quickstarts/linux-macos.md:
````diff
@@ -0,0 +1,67 @@
+
+2. Running the provided command-line command to download and adjust the permissions for the CLI:
+
+```text
````
Use bash syntax highlighting perhaps?

```suggestion
```bash
```

BTW - should we prefix commands with the `$` prompt or not?

khorne3 replied:
Yes to both

vlad-alexandru-ionescu commented on quickstarts/linux-macos.md:
````diff
@@ -0,0 +1,67 @@
+https://www.shiftleft.io/apps/HelloShiftLeft?organizationId=d64f2e4d-0d32-4c0b-bdae-891f693e2399
+```
+
+Per the instructions printed to the Terminal/Bash, open up the URL provided after 5-10 minutes have elapsed. This brings you to the Dataflows page; in the top right-hand corner, click Vulnerabilities to see the issues present in your code.
````
Perhaps this URL should take you straight to vulnerabilities view. Mind opening an issue in Product, for @mirho , to change the default landing for this URL?

khorne3 replied:
https://github.com/ShiftLeftSecurity/product/issues/3536

vlad-alexandru-ionescu commented on quickstarts/windows.md:
````diff
@@ -0,0 +1,127 @@
+
+```text
+sl auth --org "YOUR_ORG_ID" --token "YOUR_ACCESS_TOKEN"
+```
````
This step would be done after the installation. Perhaps this should be moved further down the page.

khorne3 commented:
Hey @vlad-alexandru-ionescu -- I've added your suggestions, updated the navigation a bit, and added some context + quickstart links to the homepage. If you have no additional comments, I think this is ready to go.
